python/across_api/across/models.py

```diff
@@ -0,0 +1,22 @@
+from ..base.models import DynamoDBBase  # type: ignore
+
+
+class UserModel(DynamoDBBase):
+    """
+    Represents a user in the system.
+
+    Attributes
+    ----------
+    username: str
+        The username of the user.
+    api_key: str
+        The API key associated with the user.
+    userlevel: int
+        The user level. Defaults to 1.
+    """
+
+    __tablename__ = "acrossapi_users"
+
+    username: str
+    api_key: str
+    userlevel: int = 1
```

Comment:
Remove... won't apply to OIDC.

Reply:
OK, I will just pull out all the auth code.

Reply:
Fixed by cdf0e9f